src/Common/src/Interop/Unix/System.Native/Interop.GetSystemTimeAsTicks.cs

```diff
@@ -10,7 +10,7 @@ internal static partial class Interop
 {
     internal unsafe partial class Sys
     {
-        [DllImport(Interop.Libraries.CoreLibNative, EntryPoint = "CoreLibNative_GetSystemTimeAsTicks")]
+        [DllImport(Interop.Libraries.SystemNative, EntryPoint = "SystemNative_GetSystemTimeAsTicks")]
```

Comment:
It doesn't look like there is a new package for this. I will revert this for now